src/components/select/editForm/Select.edit.display.js

```diff
@@ -15,4 +15,12 @@ export default [
       ],
     },
   },
+  {
+    weight: 1230,
+    type: 'checkbox',
+    label: 'Unique values',
+    tooltip: 'Display only unique dropdown values.',
```

Comment:
This should really say "Unique Options" so that it is not confused with Unique validation which validates the value is unique.  I would like to propose that we change anywhere it says "values" with "options", so the property will be called ```uniqueOptions``` as well.